ArduPlane/ArduPlane.cpp

```diff
@@ -39,6 +39,7 @@ const AP_Scheduler::Task Plane::scheduler_tasks[] = {
     SCHED_TASK(update_flight_mode,    400,    100),
     SCHED_TASK(stabilize,             400,    100),
     SCHED_TASK(set_servos,            400,    100),
+	SCHED_TASK(update_volz,           400,    100),
```

Comment:
Why not run the update call at the tail end of `set_servos(void)` (see ArduPlane/servos.cpp). It has the advantage of keeping all the servo output code lumped into one function, and ensures that the volz servos are scheduled immediately when all other servos are scheduled (and updated)

Reply:
Just one issue about it that @magicrub raised is that sending all channels at 400hz will need 345600 bits/sec, so i was about to change VOLZ to 50hz since these are only actuators not high speed ESCs ... what do you think?